lib/record_store/provider/dnsimple.rb

```diff
@@ -19,7 +19,7 @@ def add(record)
 
       if record.type == 'ALIAS'
         txt_alias = retrieve_current_records.detect do |rr|
-          rr.type == 'TXT' && rr.fqdn == record.fqdn && rr.txtdata == "ALIAS for #{record.cname.gsub(/.\z/, '')}"
+          rr.type == 'TXT' && rr.fqdn == record.fqdn && rr.txtdata == "ALIAS for #{record.alias.gsub(/.\z/, '')}"
```

Comment:
this pattern actually removes the last character... not just a trailing `.`

we should escape the `.` as well